android/KMAPro/kMAPro/src/main/res/menu/overflow_menu.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  tools:context=".MainActivity">
+
+      <item
+        android:id="@+id/action_text_size"
+        android:title="@string/action_text_size"
+        app:showAsAction="never"
+        android:icon="@drawable/ic_light_action_textsize" />
+
+      <item
+        android:id="@+id/action_clear_text"
+        android:title="@string/action_clear_text"
+        app:showAsAction="never"
+        android:icon="@drawable/ic_light_action_trash" />
+
+      <item
+        android:id="@+id/action_info"
+        android:title="@string/action_info"
+        app:showAsAction="never"
+        android:icon="@drawable/ic_light_action_info" />
+
+      <item
+        android:id="@+id/action_get_started"
+        android:title="@string/get_started"
+        app:showAsAction="never"
+        android:icon="@drawable/ic_action_get_started" />
+
+      <item
+        android:id="@+id/action_settings"
+        android:title="@string/action_settings"
+        app:showAsAction="never"
+        android:icon="@drawable/ic_settings" />
+
+      <item
+        android:id="@+id/action_update_keyboards"
+        android:title="@string/action_install_updates"
+        app:showAsAction="never"
+        android:icon="@drawable/ic_cloud_download"/>
```

Comment:
Doesn't this `action_update_keyboards` item also need to get added to menu-land/main.xml and menu-sw600dp/main.xml?

Reply:
Yes I started working on the landscape mode. That's not finished yet.

In the smartphone mode you see the update count at the overflow item and the install update is hidden when no updates are available.

I would like to get a feedback about the workflow with notification, update check and indicator for smartphone first.